packages/actions/index.tsx

```diff
@@ -18,23 +19,45 @@ export default function Actions({
   return (
     <div className={classNames(s.actions, s[layout])} data-testid="actions">
       {ctas.map((cta, index) => {
-        return (
-          <Button
-            key={index}
-            title={cta.title}
-            linkType={
-              cta.variant === 'tertiary-neutral' ? 'inbound' : undefined
-            }
-            url={cta.url}
-            onClick={cta.onClick}
-            size={size}
-            theme={{
-              brand: theme,
-              variant: cta.variant || 'primary',
-              background: appearance === 'dark' ? 'dark' : undefined,
-            }}
-          />
-        )
+        // The first CTA and second CTA should always
+        // have the `primary` and `secondary` variations respectively.
+        // This const is confusingly named because `Button` and `StandaloneLink`
+        // use different prop names to trigger variations (`variant` and `theme`).
+
+        const themeOrVariant = index === 0 ? 'primary' : 'secondary'
+        const isStandaloneLink = cta?.href && cta?.children
+        const isButton = cta?.title && cta?.url
+
+        if (isButton) {
+          return (
+            <Button
+              // eslint-disable-next-line react/no-array-index-key
+              key={index}
+              title={cta.title}
+              linkType={undefined}
+              url={cta.url}
+              onClick={cta.onClick}
+              size={size}
+              theme={{
+                brand: theme,
+                variant: themeOrVariant,
+                background: appearance === 'dark' ? 'dark' : undefined,
+              }}
+            />
+          )
+        }
+
+        if (isStandaloneLink) {
+          return (
+            <StandaloneLink
+              href={cta.href}
+              theme={themeOrVariant}
+              appearance={appearance}
```

Comment:
Will also want to pass onClick handler here similar to the button.

```tsx
onClick={cta.onClick}
```

Reply:
Ah, yes. Done. 👍

Reply:
Ah, yes. Done 👍